settings-test-crosspost.json

```diff
@@ -0,0 +1,30 @@
+{
+  "public": {
+    "forumType": "EAForum",
```

Comment:
We leave the forum type as `EAForum`, as the `LessWrong` forum type isn't setup for E2E testing and is likely to introduce more headaches (auth for example)